platform/broadcom/docker-syncd-brcm/Dockerfile.j2

```diff
@@ -19,7 +19,9 @@ debs/{{ deb }}{{' '}}
 ## TODO: add kmod into Depends
 RUN apt-get install -f kmod
 
-COPY ["files/dsserve", "files/bcmcmd", "start.sh", "/usr/bin/"]
+RUN apt-get install socat
```

Comment:
add into base image.

Reply:
Even for other platforms?

---
In reply to: [184841809](https://github.com/Azure/sonic-buildimage/pull/1657#discussion_r184841809) [](ancestors = 184841809)